core/benches/sigverify_stage.rs

```diff
@@ -171,18 +172,21 @@ fn bench_sigverify_stage(bencher: &mut Bencher) {
         }
         let mut received = 0;
         trace!("sent: {}", sent_len);
+        let mut messages = vec![];
         loop {
-            if let Ok((mut verifieds, _)) = verified_r.recv_timeout(Duration::from_millis(10)) {
-                while let Some(v) = verifieds.pop() {
+            if let Ok(message) = verified_r.recv_timeout(Duration::from_millis(10)) {
+                let (verifieds, _) = &*message;
+                for v in verifieds.iter().rev() {
                     received += v.len();
-                    batches.push(v);
                 }
+                messages.push(message);
```

Comment:
why this change?

Reply:
same as this reason: https://github.com/solana-labs/solana/pull/29196#discussion_r1053257145

i omitted in-source commenting here because given that this is test code and given the fact `Arc`s here. the new code here isn't so subtle.